### PR TITLE
Implemented team switching on the session view

### PR DESCRIPTION
**Pull request by Laugslander**

### CODE REVIEW

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: Soccar [UI]/src/nl/socnet/message/handler/SwitchTeamMessageHandler.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        Object controller = Main.getInstance().getController();
+        if (controller != null && controller instanceof SessionViewFXMLController) {
+            SessionViewFXMLController view = (SessionViewFXMLController) controller;
+            Platform.runLater(view::setRoomInfo);
+        }
+    }
+
+    @Override
+    protected void encode(Connection connection, SwitchTeamMessage message, ByteBuf buf) throws Exception {
```
Comment (ghost) [inline Soccar [UI]/src/nl/socnet/message/handler/SwitchTeamMessageHandler.java:47]: MAJOR Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1186)

File: Soccar [UI]/src/nl/socnet/message/SwitchTeamMessage.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.SWITCH_TEAM_MESSAGE_ID, handler = SwitchTeamMessageHandler.class)
+public final class SwitchTeamMessage extends Message {
+
+    private final String username;
+    private final TeamColour team;
+
+    public SwitchTeamMessage(String username, TeamColour team) {
```
Comment (ghost) [inline Soccar [UI]/src/nl/socnet/message/SwitchTeamMessage.java:19]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [UI]/src/nl/socnet/message/SwitchTeamMessage.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        this.username = username;
+        this.team = team;
+    }
+
+    public SwitchTeamMessage() {
```
Comment (ghost) [inline Soccar [UI]/src/nl/socnet/message/SwitchTeamMessage.java:24]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 3 issues
* MAJOR 1 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Review by BeeHiveJava — APPROVED** [review]

**Pull request merged**